### PR TITLE
navBar

### PR DESCRIPTION
**Pull request by akramkazmi71**

### CODE REVIEW

File: src/components/navBarItems.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        title: 'Result',
+        url: '/result',
+        cName: 'navLinks'
+    },
+    {
+        title: 'Contributors',
+        url: '/contributors',
+        cName: 'navLinks'
+    }
+]
```
Comment (niranjanhegde144) [inline src/components/navBarItems.js:27]: what does this do?

File: src/components/navBar.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import { navItems } from './navBarItems'
+import '../App.css'
```
Comment (niranjanhegde144) [inline src/components/navBar.js:3]: You do not need import "../App.css" here. It's imported in App.js so

File: src/components/navBarItems.js (hunk 1 above)
Comment (akramkazmi71) [reply src/components/navBarItems.js:27]: Using this we can modify the number of tabs in the nav bar, without changing the code in navBar.js. Just like JSON object, like what you did with contributors.

File: src/components/navBar.js (hunk 2 above)
Comment (akramkazmi71) [reply src/components/navBar.js:3]: So I dont have to import it in other files?
Then how will it take the css?

File: src/components/navBarItems.js (hunk 1 above)
Comment (niranjanhegde144) [reply src/components/navBarItems.js:27]: Okay

File: src/components/navBar.js (hunk 2 above)
Comment (niranjanhegde144) [reply src/components/navBar.js:3]: you are using NavBar component in App.js. Whatever component that is being used in App.js gets the CSS of App.css. So I have written comments in App.css as to where to write the CSS for the particular Component.s

Comment (akramkazmi71) [reply src/components/navBar.js:3]: okay

**Pull request merged**